### ISSUE TITLE
What does this package actually do?

### ISSUE THREAD

**Issue by rjmunro** [Blackshawk/behat-atom#7]
Does it do anything other than syntax colour .feature files (which are written in the gherkin language)?

Does it have any advantages over https://atom.io/packages/language-gherkin

Please expand the README with a bit more detail.

It would be great if it could run `behat -dl` and use that for completion.

**Comment by mccrodp** [comment]
Nice to have syntax highlighting, but was hoping for clicking through from `.feature` steps to `FeatureContext.php` functions somehow, would have been nice, guess that's what IDE's are for.

**Comment by nettles-jarrod** [comment]
Hey guys, I created this package a day or two after Atom was released. Its almost a line-for-line port of the Textmate Behat bundle that does the same thing. It was a quick attempt to 1) test out the Atom plugin system and 2) give myself a useful feature for work I was doing at the time.

Unfortunately I'm too busy to keep this package up to date (I don't even use Atom anymore) or especially to add new features. If someone is interested, I'm more than willing to grant write access to the repo.

Cheers!
